docs/jazzy.yml

```diff
@@ -170,6 +173,13 @@ custom_categories:
       - NavigationViewportDataSource
       - ViewportDataSourceType
       - ViewportDataSourceDelegate
+      - FollowingCameraOptions
```

Comment:
Should these items be sorted alphabetically?

Reply:
I don't think that we have such requirement here.